#Control message for the mesh network
uint8 windowSize
